src/Map.jsx

```diff
@@ -33,7 +34,16 @@ class Map extends Component {
         minPitch: 0,
         maxPitch: 85,
       },
+      coordinates: [0, 0],
+      info: '',
     };
+
+    // bind this to onMarker method for child component' access to this.state
+    this.onMarkerClick = this.onMarkerClick.bind(this);
+  }
+
+  onMarkerClick(x, y, info) {
```

Comment:
what are x and y? since we're using real coordinates, why not use `lat` and `lon`?

Reply:
alright, makes sense